pkg/provider/cloud_test.go

```diff
@@ -58,41 +62,45 @@ func makeCloudConfig(kubeconfig, namespace string, loadbalancerEnabled, instance
 	}
 }
 
-func TestNewCloudConfigFromBytes(t *testing.T) {
-	tests := []struct {
-		configBytes         string
-		expectedCloudConfig CloudConfig
-		expectedError       error
-	}{
-		{minimalConf, makeCloudConfig(kubeconfig, "", true, true, 5), nil},
-		{loadbalancerConf, makeCloudConfig(kubeconfig, "", false, true, 3), nil},
-		{instancesConf, makeCloudConfig(kubeconfig, "", true, false, 5), nil},
-		{namespaceConf, makeCloudConfig(kubeconfig, ns, true, true, 5), nil},
-		{allConf, makeCloudConfig(kubeconfig, ns, false, false, 5), nil},
-	}
+var _ = Describe("Cloud config", func() {
 
-	for _, test := range tests {
-		config, err := NewCloudConfigFromBytes([]byte(test.configBytes))
-		if !reflect.DeepEqual(config, test.expectedCloudConfig) {
-			t.Errorf("Expected: %v, got %v", test.expectedCloudConfig, config)
-		}
-		if test.expectedError != nil && err != nil && err.Error() != test.expectedError.Error() {
-			t.Errorf("Expected: '%v', got '%v'", test.expectedError, err)
-		}
-	}
-}
+	DescribeTable("Get CloudConfig from Bytes", func(configBytes string, expectedCloudConfig CloudConfig, expectedError error) {
+		config, err := NewCloudConfigFromBytes([]byte(configBytes))
+		Expect(config).To(Equal(expectedCloudConfig))
+		Expect(err).To(BeNil())
+	},
+		Entry("With Minimal Configuration", minimalConf, makeCloudConfig(infraKubeConfigPath, "", true, true, 5), nil),
```

Comment:
I don't think we need to have every word uppercase, could you adjust to instancesV2 tests style?